### DISCUSSION TITLE
Repeating tasks

### DISCUSSION THREAD

**Discussion by manfredlotz** [zim-desktop-wiki/zim-desktop-wiki#1894] [Q&A] [answered]
It seems there is no support for repeating tasks, like weekly, monthly aso. 

Is this planned to add? Or is there a possibility to get this done via some feature I am not aware of?

**Comment by jaap-karssenberg** [comment, marked as answer]
No it is not supported and also no concrete plan.

Only thing you can do is to a) just update the due date manually when you
check a repeating task, or b) add repeating tasks in the template for
journal pages if you are also actively using the journal.

--- Jaap

**Reply by manfredlotz** [reply]
Yes, I use journal heavily. I will take a look at templates for journal pages. Thanks a lot for this hint.
